pandas/tests/resample/test_datetime_index.py

```diff
@@ -694,6 +694,20 @@ def test_resample_timestamp_to_period(simple_date_range_series):
     tm.assert_series_equal(result, expected)
 
 
+def test_datetime_resample_sum_min_count():
+    # GH 19974
+    index = date_range(start="2018", freq="M", periods=6)
```

Comment:
this is not actually testing for the issue as this is a DTI and not a PI.

Reply:
Yes, this test was using DTI as that was referenced as the expected behaviour in the issue.
Not it's removed and the only test left is using PI - `test_period_resample_sum_min_count` in `pandas/tests/resample/test_period_index.py`.